Handle empty query argument of get_query_tokens

```
query = ''

    def get_query_tokens(query):
        """
        :type query str
        :rtype: list[sqlparse.sql.Token]
        """
        query = preprocess_query(query)
    
>       tokens = TokenList(sqlparse.parse(query)[0].tokens).flatten()
E       IndexError: tuple index out of range
```

--- a/sql_metadata.py
+++ b/sql_metadata.py
@@ -36,8 +36,13 @@ def get_query_tokens(query):
     :rtype: list[sqlparse.sql.Token]
     """
     query = preprocess_query(query)
+    parsed = sqlparse.parse(query)
 
-    tokens = TokenList(sqlparse.parse(query)[0].tokens).flatten()
+    # handle empty queries (#12)
+    if not parsed:
+        return []
+
+    tokens = TokenList(parsed[0].tokens).flatten()
     # print([(token.value, token.ttype) for token in tokens])
 
     return [token for token in tokens if token.ttype is not Whitespace]

--- a/test/test_query.py
+++ b/test/test_query.py
@@ -1,9 +1,24 @@
 from unittest import TestCase
 
-from sql_metadata import preprocess_query, get_query_columns, get_query_tables, get_query_limit_and_offset
+from sql_metadata import preprocess_query, get_query_tokens,\
+    get_query_columns, get_query_tables, get_query_limit_and_offset
+
+from sqlparse.tokens import DML, Keyword
 
 
 class TestUtils(TestCase):
+
+    def test_get_query_tokens(self):
+        assert get_query_tokens("") == []
+
+        tokens = get_query_tokens("SELECT * FROM foo")
+
+        assert len(tokens) == 4
+
+        assert tokens[0].ttype is DML
+        assert str(tokens[0]) == 'SELECT'
+        assert tokens[2].ttype is Keyword
+        assert str(tokens[2]) == 'FROM'
 
     def test_preprocess_query(self):
         self.assertEquals(

